pykeepass/pykeepass.py

```diff
@@ -268,6 +268,42 @@ def entries(self):
         excluding history"""
         return self.find_entries()
 
+    @property
+    def database_name(self):
+        """Name of database"""
+        elem = self._xpath('/KeePassFile/Meta/DatabaseName', first=True)
```

Comment:
I see that in the other cases the `path` is inlined in the `_xpath` fn I would do them all the same, and if there is already a code style in this file, follow it.

Reply:
I've used recyclebin_group as base, so this is existing code style. What should i change then? There is no Coding Guidelines in this project..

Reply:
If the file is consistent in its style follow that, if not choose one style of your liking and use the same for all 4 fns.

Reply:
Done